contracts/HealthNodePatient.sol

```diff
@@ -10,19 +10,22 @@ contract PatientNode is AccessControl {
     bytes32 public constant DOCTOR_ROLE = keccak256("DOCTOR_ROLE");
     bytes32 public constant PATIENT_ROLE = keccak256("PATIENT_ROLE");
 
+//we need to add consultation note
     // Patient
     struct Patient {
         bool registered;
         string name;
         uint256 age;
         string gender;
         string ipfsRecordHash;  // Hash to store medical records from IPFS
```

Comment:
I believe the hash should only be on the backend service. Please let us have a call to sync on this.